contributors/THIRD-PARTY-EXTENSION.md

```diff
@@ -0,0 +1,104 @@
+## Introduction
+
+Welcome to the guide for developing external extensions in the `create-eth` repository. This document outlines the workflow for creating and testing your extensions.
```

Comment:
I think we need to give more context about extensions. What are they? What they allow you to do? Link to example extensions.

And maybe adding something about extensions (with a link to this page) in the main repo README